django_redis/serializers/pickle.py

```diff
@@ -1,24 +1,27 @@
 import pickle
+from typing import Any
 
 from django.core.exceptions import ImproperlyConfigured
 
 from .base import BaseSerializer
 
 
 class PickleSerializer(BaseSerializer):
-    def __init__(self, options):
+    def __init__(self, options) -> None:
         self._pickle_version = -1
         self.setup_pickle_version(options)
 
-    def setup_pickle_version(self, options):
+        super(PickleSerializer, self).__init__(options=options)
```

Comment:
As this code is Python 3 only, can use `super()` without arguments:

```suggestion
        super().__init__(options=options)
```

Reply:
Thanks, I did not know that